libsolidity/parsing/Parser.cpp

```diff
@@ -493,6 +522,13 @@ Parser::FunctionHeaderParserResult Parser::parseFunctionHeader(bool _forceEmptyN
 			else
 				result.stateMutability = parseStateMutability();
 		}
+		else if (token == Token::Override)
+		{
+			if (result.overrides)
+				parserError("Override already specified.");
+
```

Comment:
Maybe use `else`, since the user might not expect the override to be overridden?

Reply:
Ah ok sorry, this won't work because it would totally confuse the parser.